traits.yml

```diff
@@ -1,11 +1,24 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/OpenAssetIO/OpenAssetIO-TraitGen/main/python/openassetio_traitgen/schema.json
 # yamllint disable-line rule:document-start
 package: openassetio-mediacreation
-
 description: Well-known Traits and Specifications for use in OpenAssetIO
   hosts and managers.
-
 traits:
+  usage:
```

Comment:
[question] Why do we need usage traits?

Reply:
See commit note:

> We opted to add base traits for 'usage' to the package. These should be
composed with the other relevant traits when defining a specification to
allow runtime introspection of the trait set. Amongst other uses, this
helps managers in their implementation of methods such as
`managementPolicy` where they need to be able to determine the nature of
the query.

Was based on the off-line discussion we had about being able to better understand what a trait set was meant to be for.

Reply:
Yup rings a bell, thanks.